pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -86,12 +86,16 @@ fn create_token(token_name: &[u8], ticker: Ticker, keyring: Public) {
     ));
 }
 
-fn next_block() {
+pub fn next_block() {
     let block_number = System::block_number() + 1;
-    System::set_block_number(block_number);
+    set_current_block_number(block_number);
     let _ = Scheduler::on_initialize(block_number);
 }
 
+pub fn set_current_block_number(block: u64) {
```

Comment:
Hmm; what's the value of this function?

Reply:
In USD or in BTC 😉 ?

Reply:
This joke is so bad, it almost feels like one of mine xD

Reply:
I'm the only one from Gothenburg here, and we invented bad & dry jokes. ;)